internal/rules/aws/apigateway/enable_cache_test.go

```diff
@@ -0,0 +1,87 @@
+package apigateway
+
+import (
+	"testing"
+
+	defsecTypes "github.com/aquasecurity/defsec/pkg/types"
+
+	v1 "github.com/aquasecurity/defsec/pkg/providers/aws/apigateway/v1"
+
+	"github.com/aquasecurity/defsec/pkg/state"
+
+	"github.com/aquasecurity/defsec/pkg/scan"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
could you make sure the imports are sorted? `go import` should take care of it or your IDE of choice.

Reply:
current rules are not following this practice i.e. imports are not sorted. So should we keep it consistent?